Adding all file and directory content for package

metalphase opened the pull request:
Adding necessary .toml, readme, requirements files, as well as the contents of the package in src/

metalphase committed:
adding .toml, readme, and requirements files

adding all source files for package

removing /data directory

Adding data files and modifying __init__() files

removing F64601860 directory and adding test_response.py to run with pytest

removing extra requirements not needed in requirements.txt and removing unneeded code in config.py

Adding build_instructions.txt which will give instructions to the user for installing the package locally

 removing data/.DS_Store

removing build_instructions.txt and adding instructions to README.md

Delete ls

Tohuvavohu commented:
Why do you need the following pieces

/data/bright_source_cat.fits
/data/F646018360

there may be other extraneous pieces too.

You can look at what you actually need to do the relevant response calculation and tests, and only include that.

Tohuvavohu commented:
also many of the requirements listed are not needed

Tohuvavohu commented on src/nitrates_response/config.py:
```diff
@@ -0,0 +1,143 @@
+}
+
+
+# email password file
```
Dont need this

metalphase commented:
@Tohuvavohu  Removed the F64601860 directory and added the test_response.py file that will run using pytest. I also edited the requirements to just include the necessary modules and just cleaned up the repository in general according to your advice

Tohuvavohu reviewed (changes requested):
Great job, it's looking really nice!

Two things:
1) If this is set up properly, install should only require 
'pip install .' in the root directory

2) please squash your commits before first merge

Tohuvavohu commented:
Also, when you've figured that out, install instructions should be in README.md

metalphase commented:
Looks like the package should install just as you described in 1.  I've included the install instructions into the README.md file. I'm still figuring out how to squash my last commits. Should I be squashing all of my commits into a singular one after your initial commit for the repo?

Tohuvavohu commented:
Yep!

Tohuvavohu commented:
also you have an extraneous, empty, file called 'ls' which should be removed

metalphase commented:
@Tohuvavohu alright, was a nail-biter but was able to squash the commits successfully and now all the commits show up as a singular commit.

metalphase commented:
Was there any other things that needed to be changed?
